### PR TITLE
Fix postcss-editor-styles processing

### PR DESCRIPTION
**Pull request by darylldoyle**
### Description of the Change

Only load and run the `postcss-editor-styles` plugin when processing `editor-style.css`

### Alternate Designs

N/A

### Benefits

Only the `editor-style.css` file will get the `.editor-styles-wrapper` class.

### Possible Drawbacks

None

### Verification Process

Tested locally on project using wp-scaffold

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### Applicable Issues

#40 

### Changelog Entry

- Fixed issue with editor class being appended to all styles.

### CODE REVIEW

**Review by nicholasio — APPROVED** [review]
This fixes the bug for me. Good catch, this was introduced by #33.

I'll wait for @ryanwelcher approval before merging and release a new version. Want to get this out to npm asap as it's a very disruptive bug.

**Comment by ryanwelcher** [comment]
Great catch here @darylldoyle! I ran into this issue as well and came up with a slightly different fix but if this solves the issue, I'm good.

**Review by ryanwelcher — APPROVED** [review]

**Pull request merged**
